ferry.go

```diff
@@ -356,19 +377,26 @@ func (f *Ferry) RunStandaloneDataCopy(tables []*schema.Table) error {
 	dataIterator.AddBatchListener(f.BatchWriter.WriteRowBatch)
 	f.logger.WithField("tables", tables).Info("starting standalone table copy")
 
-	dataIterator.Run()
+	dataIterator.Run(context.TODO())
```

Comment:
why is this a `context.TODO`? Do we not have access to the parent context here?

Reply:
There's currently not capabilities to stop the process of the delta copier, which uses this method (which does not take in a context for the time being).